pontoon/base/views.py

```diff
@@ -1087,7 +1085,6 @@ def transvision(request, repo, title):
 
             return HttpResponse(json.dumps({
```

Comment:
I considered doing it now that "title" is removed, but this format is consistent with other similar (Machinery) views.